lib/Widgets/Utils.vala

```diff
@@ -130,6 +130,10 @@ public static string accel_to_string (string accel) {
             ///TRANSLATORS: This is a non-symbol representation of the "+" key
             arr += _("Plus");
             break;
+        case Gdk.Key.KP_Enter:
```

Comment:
I'm not sure we want to lump this in, do we? Right now it shows "Enter (keypad)". It sounds like for the most part most apps don't distinguish, but there are some apps (like [Pro Tools](https://ux.stackexchange.com/questions/54461/should-i-use-the-name-enter-or-return-key)) which do use the keys for different things.

Reply:
@danrabbit I was just being consistent with the plus/minus lines above. But I am fine to remove it in this PR and correct those later (if we want).